test/cosmoz-bottom-bar-view.html

```diff
@@ -51,67 +51,131 @@
 	(function () {
 		'use strict';
 
-		var flushBottomBar = function (bottomBar, done){
-			var _observer = bottomBar._nodeObserver;
+		let flushBottomBar = (bottomBar, done) => {
+			let _observer = bottomBar._nodeObserver;
 			if (_observer && _observer.flush === 'function') {
 				_observer.flush();
 			}
 			flush(done);
 		};
 
-		suite('<cosmoz-bottom-bar-view>', function () {
-			var
+		suite('<cosmoz-bottom-bar-view>', () => {
+			let
 				bottomBarView,
 				bottomBar;
 
-			setup(function (done) {
+			setup(done => {
 				bottomBarView = fixture('bottomBarViewFixture');
 				bottomBar = bottomBarView.$.bottomBar;
-				flushBottomBar(bottomBar, function (){
+				flushBottomBar(bottomBar, () => {
 					Polymer.Base.async(done, 50);
 				});
 			});
 
-			test('bottomBar is visible', function (done) {
+			test('bottomBar is visible', done => {
 				assert.isTrue(bottomBar.visible);
 				done();
 			});
 		});
 
-		suite('scrolling bottom-bar-view', function () {
-			var
+		suite('scrolling bottom-bar-view', () => {
+			let
 				bottomBarView,
 				bottomBar;
 
-			setup(function (done) {
+			setup(done => {
 				bottomBarView = fixture('scrollingBottomBarViewFixture');
 				bottomBar = bottomBarView.$.bottomBar;
-				flushBottomBar(bottomBar, function (){
+				flushBottomBar(bottomBar, () => {
 					Polymer.Base.async(done, 50);
 				});
 			});
 
-			test('bottomBar is hidden when scrolling down', function (done) {
+			test('bottomBar is hidden when scrolling down', done => {
 				bottomBarView.$.scroller.scrollTop = 20;
 				// Mutation observer is async, so wait before testing.
-				Polymer.Base.async(function () {
+				Polymer.Base.async(() => {
 					assert.isFalse(bottomBar.visible);
+					bottomBarView.$.scroller.scrollTop = 0;
```

Comment:
Design the test cases so they reset what they need themselves in the beginning and then adjust accordingly async or whatever. Right now we're resetting the tests at the end of the previous test which makes it less clear and the tests depend on eachother.

Reply:
I have updated the tests in bottom-bar-view.html. But the tests in bottom-bar.html has started to fail.

```
chrome 70                ✖ test/cosmoz-bottom-bar.html » bottomBarWithOverflowingButton » menu button should be visible
  expected true to be false
```

I am not sure how to debug this, since I cannot manage to run the yarn tests locally. If I do `export SAUCE_ACCESS_KEY=<key>; export SAUCE_USERNAME=nomego; yarn test` then I get this:

```
Sauce tunnel failed:
Error: Sauce Connect could not establish a connection.
```